coalib/bears/Bear.py

```diff
@@ -26,6 +26,11 @@
 
 class Debugger(pdb.Pdb):
 
+    def __init__(self, *args, **kwargs):
+        self.bear = kwargs.get('bear', None)
+        kwargs.pop('bear', None)
```

Comment:
why not just declaring an optional argument `bear=None`? Then you don't need to pop it off from `kwargs`